dist/gcc-compatible/Makefile.include

```diff
@@ -1,5 +1,5 @@
 USER_TARGET=libevercrypt.a
-USER_CFLAGS=-march=native -mtune=native -Wno-unused
+#USER_CFLAGS=-march=native -mtune=native -Wno-unused
```

Comment:
this is also an auto-generated file, so we'd have to remove these flags from the main Makefile (line 645), then tweak Makefile.tmpl to say something like:

```
# Neither the main Makefile or the configure script specified -march, default to native
ifeq ($(subst -march,,$(USER_CFLAGS)$(CFLAGS)),$(USER_CFLAGS)$(CFLAGS))
CFLAGS += -march=native -mtune=native
```
```